parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -16,7 +16,7 @@ SECURE_TLS_BOOTSTRAP_KUBELET_EXEC_PLUGIN_DOWNLOAD_DIR="/opt/azure/tlsbootstrap"
 SECURE_TLS_BOOTSTRAP_KUBELET_EXEC_PLUGIN_VERSION="v0.1.0-alpha.2"
 TELEPORTD_PLUGIN_DOWNLOAD_DIR="/opt/teleportd/downloads"
 TELEPORTD_PLUGIN_BIN_DIR="/usr/local/bin"
-CONTAINERD_WASM_VERSIONS="v0.3.0 v0.5.1 v0.8.0"
```

Comment:
Do we have any users on the older versions of the shim, say 0.3.0? By removing the older versions, are we going to break their existing workflows? If so, should we be more cautious about deprecating older versions and probably notife customers beforehand? 

I am fine with deprecation in general but I prefer putting a notification to customers first.

Reply:
These older versions are getting a 404 on downloads during the VHD builds. Customers won't be affected/already must be affected in that case right?

Reply:
Only case I can think of is users migrating from an older version working earlier to a new nodepool/VHD image that has the newer version. What would the expected behavior for that be, before this change (as that path existed, because the older version was not available for a while)?

Reply:
For the notification to users, adding a note in the AKS release notes should be sufficient right?

Reply:
The spin and slight are downloading successfully for v0-3-0 and v0-5-1, but not wws, which makes me think that wws isn't available for those.
 
Is it possible they are still using spin / slight for v0-3-0 and v0-5-1 without wws? @Mossaka 

If not, then I don't think any customers would have been able to use those versions for quite some time.

Reply:
> Is it possible they are still using spin / slight for v0-3-0 and v0-5-1 without wws?

Yes, we should remove wws from v0.3.0 and v0.5.0 because it wasn't introduced for these versions

Reply:
maybe what you need is a condition logic saying that if the version if 0-3-0 or 0-5-0, don't install wws shim

Reply:
Alrighty thanks!

Reply:
@Mossaka Good morning! I have implemented your suggestions, adding the v3 and v5 versions back in the baker.go/baker_test.go files and creating the conditional in cse_install. Would you mind giving this PR another look?